extensions/interactions/DragAndDropSortInput/directives/drag-and-drop-sort-input-validation.service.ts

```diff
@@ -109,8 +108,8 @@ export class DragAndDropSortInputValidationService {
       var noOfMismatches = 0;
       var inputs = earlierRule.inputs.x;
       var answer = laterRule.inputs.x;
-      for (var i = 0; i < math.min(inputs.length, answer.length); i++) {
-        for (var j = 0; j < math.max(answer[i].length, inputs[i].length);
+      for (var i = 0; i < min(inputs.length, answer.length); i++) {
+        for (var j = 0; j < max(answer[i].length, inputs[i].length);
```

Comment:
Why not use the JS `min` and `max` methods here?

Reply:
Done!